app/lib/server/server.go

```diff
@@ -144,10 +146,17 @@ func InteractiveConfig(binaryName string) {
 
 	// create the keys
 	privStr, pubStr := createKeyPair()
+	ips, err := net.LookupIP(publicAddress.Host())
+	log.ErrFatal(err)
+	cert, key, err := network.NewCertKey(network.NewTLSCert(big.NewInt(1), "ch", "epfl", "dedis",
```

Comment:
Well our goal is more the end user than our tests, I think it's our job to make the two work together. But anyway, we already started with user inputs for the addresses, don't you think it's a bit weird to have both user inputs value + user options at the same time for a one-time operations ? I do..  Moreover, a cert a many fields and having a looooong command line with many options seems rather unelegant also 
```
./cothorityd setup -DNSName "blabla" -owner "me" -address "here" -org "none for the moment" -Unit "blabla" ....
...
## USER INPUT FOR IP ADDRESS ##
```

I prefer having our test being not elegant that our command line. what do you think ?